build.sbt

```diff
@@ -5,6 +5,8 @@ version := "2.0.1-SNAPSHOT"
 licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
 
 sbtPlugin := true
+crossSbtVersions := Seq("1.8.2", "1.4.9") // Used for checking compatibility with sbt 1.4+
```

Comment:
The plugin no longer works on sbt 1.3, but still does on 1.4 (1.4 is when there was a major rewrite of the incremental compiler, which broke `KotlinTest`). We can execute the scripted tests with both 1.4.9 and the current version to check for compatibility.

Reply:
👍